packages/webarchitect/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@canonical/webarchitect",
+  "description": "A tool to test the compliance with architecture specifications for packages and applications.",
+  "version": "0.9.0-experimental.12",
+  "type": "module",
+  "module": "dist/esm/index.js",
+  "types": "dist/types/index.d.ts",
+  "bin": {
+    "webarchitect": "dist/esm/cli.js"
+  },
+  "files": ["dist", "rulesets"],
+  "author": {
+    "email": "webteam@canonical.com",
+    "name": "Canonical Webteam"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/canonical/ds25"
+  },
+  "license": "LGPL-3.0",
```

Comment:
Double check our licensing page, I think this might be intended as a standalone tool so it needs GPL3

https://library.canonical.com/legal/licensing-policy